src/addressindex.h

```diff
@@ -297,13 +294,15 @@ struct CMempoolAddressDeltaKey
     uint256 txhash;
     unsigned int index;
     int spending;
+    bool invite;
 
-    CMempoolAddressDeltaKey(int addressType, uint160 addressHash, uint256 hash, unsigned int i, int s) {
+    CMempoolAddressDeltaKey(int addressType, uint160 addressHash, uint256 hash, unsigned int i, int s, bool is_invite) {
         type = addressType;
         addressBytes = addressHash;
         txhash = hash;
         index = i;
         spending = s;
+        invite = is_invite;
     }
 
     CMempoolAddressDeltaKey(int addressType, uint160 addressHash) {
```

Comment:
you should set invite here too

Reply:
only place it is used is here
https://github.com/meritlabs/merit/blob/59a402867c267a338f907dc67f4ff8132223f9ea/src/txmempool.cpp#L536
and it doesn't matter if it's an invite or not